constants/util.go

```diff
@@ -21,7 +21,7 @@ var AvailableAssemblers = map[string]*AssemblerDetails{
 		Name:    MegaHit,
 		DHubURL: "docker.io/vout/megahit", // https://github.com/voutcn/megahit
 		Comm: func(i, o string) []string {
-			return []string{"--test"} // TODO: fill in appropriate params for MegaHit
+			return []string{"-r","/raw_sequence_input.fastq", "-o", "/output/genomagic-megahit_output"}
```

Comment:
What do you think about leaving a comment, above `return`, about the params that are passed to MegaHit? I think it will help in the future since it's not something that inside our codebase- it's something external that is called.